packages/core_extended/core_extended.112.35.00/files/openbsd-quota-disable.diff

```diff
@@ -0,0 +1,64 @@
+diff --git a/lib/extended_unix_stubs.c b/lib/extended_unix_stubs.c
+index 40b4d10..cbede42 100644
+--- a/lib/extended_unix_stubs.c
++++ b/lib/extended_unix_stubs.c
+@@ -12,7 +12,10 @@
+ #include <string.h>
+ #include <time.h>
+ #include <sys/statvfs.h>
++#if (defined _LINUX_QUOTA_VERSION)
```

Comment:
Ah, no, I'd essentially forgotten about it. I think doing it "properly" will be enough work that it's not a great idea to shoot for this release.